### DISCUSSION TITLE
Why can't i make my repository private?

### DISCUSSION THREAD

**Discussion by aznoranton** [community/community#117035] [Repositories] [answered]
### Select Topic Area

Question

### Body

Why can't i make my repository private ? I get this error "Due to restrictions on this repository, you cannot change the visibility." however i am the only owner of my repository and not the member of any organization. Can't find any solution that helps

**Comment by AlexanderFRT** [comment, marked as answer]
A couple of reasons below, my best advice is just be patient and wait.

**1. Repository Transfer:** If the repository was previously transferred from another user or organization, there may be lingering restrictions or permissions set by the previous owner that are preventing the user from changing the visibility.

**2. GitHub Restrictions:** GitHub occasionally imposes restrictions on certain actions or repositories, particularly if there are suspected policy violations or security concerns associated with the account or repository.

**3. Pending Actions:** If there are any pending actions, such as ongoing repository transfers (point 1) or changes initiated by the user or GitHub itself, it may temporarily restrict certain actions, including changing the visibility of the repository. 

Either wait for these actions to complete or reach out to GitHub support, so they help you to resolve this issue.

**Comment by Sohailkhan2k02** [comment]
After some investigation, I've found that there are several possible reasons for this error message, including:

Repository Restrictions: GitHub imposes certain restrictions on repositories, particularly those associated with specific types of projects or actions. These restrictions might prevent the visibility settings from being changed in certain cases.

Organization Membership: Although I'm not currently a member of any organization, there might be residual permissions or settings from a previous organization membership that are affecting the repository visibility. I'll double-check to ensure that there are no lingering organization affiliations.

Support Inquiry: To resolve this issue, I'll reach out to GitHub Support for further assistance. They can provide insights into any underlying factors affecting the repository visibility settings and offer guidance on how to proceed.
